sdk/include/psdk/shobjidl.idl

```diff
@@ -120,6 +120,12 @@ cpp_quote("#include <prsht.h>")
 cpp_quote("typedef LPFNADDPROPSHEETPAGE LPFNSVADDPROPSHEETPAGE;")
 cpp_quote("#endif")
 
+cpp_quote("typedef struct _SFVM_PROPPAGE_DATA {")
+cpp_quote("    DWORD                dwReserved;")
+cpp_quote("    LPFNADDPROPSHEETPAGE pfn;")
+cpp_quote("    LPARAM               lParam;")
+cpp_quote("} SFVM_PROPPAGE_DATA;")
```

Comment:
In official headers, this struct is _NOT_ in the idl file (and the autogenerated shobjidl.h), but is only in shlobj.h header. Please move it there instead.

Reply:
Moved to `<shlobj.h>`.